### DISCUSSION TITLE
How do I build project64 using the source code? I'm a beginner to coding.

### DISCUSSION THREAD

**Discussion by madame-froglette** [project64/project64#2321] [Support]
I recently installed the emulator, but I ran into a common bug in a game I want to play. The solution to the bug involves editing the source code. I uninstalled the emulator, and I've downloaded the files and edited the document I need to. However, I've no idea what my next steps are. How do I install the emulator with the edits I made using vscode? I am very much a beginner to all this. I just want to play Paper Mario :(

Thanks so much!

**Comment by project64** [comment]
what bug requires modifying code?

**Reply by madame-froglette** [reply]
The same one from this thread: https://github.com/project64/project64/issues/2218

**Comment by project64** [comment]
that has already been fixed in the current version

**Reply by madame-froglette** [reply]
It must be a similar problem, then.  It happens in the beginning of the game in the middle of a battle with some goombas, when I try to use headbonk. IDK what the deal is :(
![GLideN64_PAPER_MARIO_000](https://user-images.githubusercontent.com/120063794/206828026-9a66f582-e17a-4feb-9af1-97fd39f7096a.png)

**Reply by project64** [reply]
what version are you using

**Reply by madame-froglette** [reply]
3.0.1.5664

**Reply by project64** [reply]
use version 4
https://www.pj64-emu.com/nightly-builds
